pkg/cloud/gcp/actuators/machine/reconciler_test.go

```diff
@@ -4,14 +4,12 @@ import (
 	computeservice "github.com/openshift/cluster-api-provider-gcp/pkg/cloud/gcp/actuators/services/compute"
 	corev1 "k8s.io/api/core/v1"
 
-	"testing"
-
 	"fmt"
-
 	gcpv1beta1 "github.com/openshift/cluster-api-provider-gcp/pkg/apis/gcpprovider/v1beta1"
 	machinev1beta1 "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	controllerfake "sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"testing"
```

Comment:
This should be grouped with "fmt" (i.e., all the stdlib imports are grouped together).

Reply:
@frobware I don't disagree, but go fmt does :/  Let me see if I'm doing something wrong.

Reply:
gofmt won't reorder imports. That's why I created https://jira.coreos.com/browse/CLOUD-507